src/account/utils/Types.ts

```diff
@@ -208,6 +208,7 @@ export type BuildUserOpOptions = {
   dummyPndOverride?: BytesLike
   useEmptyDeployCallData?: boolean
   useExecutor?: boolean
+  signatureOverride?: Hex
```

Comment:
this would be useful for gas estimation from bundler?
as in when calling eth_estimateUserOpGas

Reply:
Yes but will remove this for now to simplify the PR, is not needed now, it was making things easier when testing modules with multi signature needed.

Reply:
ok lets take a note.